packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -131,39 +127,39 @@
       > .label-text {
         color: var(--terra-form-checkbox-disabled-font-color, #9b9fa1);
         cursor: default;
-        opacity: var(--terra-form-checkbox-disabled-font-opacity, none);
+        opacity: var(--terra-form-checkbox-disabled-font-opacity, 1);
 
         &::before {
-          background-color: var(--terra-form-checkbox-disabled-background-color, none);
+          background-color: var(--terra-form-checkbox-disabled-background-color, transparent);
           background-image: var(--terra-form-checkbox-disabled-background-image, none);
-          border-color: var(--terra-form-checkbox-disabled-border-color, none);
+          border-color: var(--terra-form-checkbox-disabled-border-color, transparent);
           cursor: default;
         }
       }
 
       &:hover > .label-text::before {
-        border-color: var(--terra-form-checkbox-disabled-hover-border-color, none);
-        border-width: var(--terra-form-checkbox-disabled-hover-border-width, none);
+        border-color: var(--terra-form-checkbox-disabled-hover-border-color, transparent);
+        border-width: var(--terra-form-checkbox-disabled-hover-border-width, 0);
       }
     }
     /* stylelint-enable max-nesting-depth */
 
     /* Focus custom checkbox styling */
     .native-input:focus + .label-text::before {
-      border-color: var(--terra-form-checkbox-focus-border-color, none);
-      border-width: var(--terra-form-checkbox-focus-border-width, none);
+      border-color: var(--terra-form-checkbox-focus-border-color, transparent);
+      border-width: var(--terra-form-checkbox-focus-border-width, 0);
     }
 
     /* Large focus ring */
     .native-input:focus + .label-text::after {
-      background-color: var(--terra-form-checkbox-focus-ring-background-color, none);
+      background-color: var(--terra-form-checkbox-focus-ring-background-color, transparent);
       border-radius: 50%;
       content: '';
-      height: var(--terra-form-checkbox-focus-ring-height, none);
-      left: var(--terra-form-checkbox-focus-ring-left, none);
+      height: var(--terra-form-checkbox-focus-ring-height);
+      left: var(--terra-form-checkbox-focus-ring-left);
       position: absolute;
-      top: var(--terra-form-checkbox-focus-ring-top, none);
-      width: var(--terra-form-checkbox-focus-ring-width, none);
+      top: var(--terra-form-checkbox-focus-ring-top);
+      width: var(--terra-form-checkbox-focus-ring-width);
```

Comment:
Do these just dissolve when they are left empty or do they get added as invalid and fall back to browser defaults?

Reply:
They get added as is, `width: var(--terra-form-checkbox-focus-ring-width);`, in the extracted CSS file that runs through all the loaders. In browsers that don't support CSS custom properties, it will skip over this style as the browser views it as an invalid property and continues to use the browser default style for width on checkbox inputs. 

In browsers that support CSS custom properties, we are relying on the fact that in the default theme, we don't set a value for `--terra-form-checkbox-focus-ring-width`, making it essentially undefined allowing the browser to continue to use the default style for width on checkbox inputs.

This is an approach we'll likely start using more often where components have styles that are needed for other themes, but not used in the default theme.